website/versioned_docs/version-24.9/Configuration.md

```diff
@@ -1095,6 +1095,8 @@ A map from regular expressions to paths to transformers. A transformer is a modu
 
 Examples of such compilers include [Babel](https://babeljs.io/), [TypeScript](http://www.typescriptlang.org/) and [async-to-gen](http://github.com/leebyron/async-to-gen#jest).
```

Comment:
I guess this kinda belongs together, so given that this section is already a big collection of paragraphs maybe let's do:
```
Examples of such compilers include [Babel](https://babeljs.io/), [TypeScript](http://www.typescriptlang.org/) and [async-to-gen](http://github.com/leebyron/async-to-gen#jest). If you need more advanced functionality or want to build a transformer with babel support, please visit our [Custom Transformer](TutorialReact.md#custom-transformers) section.
```
You also inserted it in different places across the versions (e.g. before "You can pass" here but afterwards in `docs`), make sure that's consistent please

Reply:
ok, thoughts on making it into a list rather than having a big blob of description ? 

Examples of such compilers include:
- [Babel](https://babeljs.io/)
- [TypeScript](http://www.typescriptlang.org/)
- [async-to-gen](http://github.com/leebyron/async-to-gen#jest)

To build your own please visit the [Custom Transformer](TutorialReact.md#custom-transformers) section.

**Versus** 

Examples of such compilers include [Babel](https://babeljs.io/), [TypeScript](http://www.typescriptlang.org/) and [async-to-gen](http://github.com/leebyron/async-to-gen#jest). If you need more advanced functionality or want to build a transformer with babel support, please visit our [Custom Transformer](TutorialReact.md#custom-transformers) section.

Reply:
Actually, reading through the paragraph above this, that one already says example twice, looks like it has grown over time with no one paying attention to overall coherence - maybe rewrite that and integrate this paragraph into it?

Reply:
hey @jeysal, just got around to updating it. Please let me know what you think!

Reply:
Bit sad you didn't pick up the suggestion of reducing the "example example example" repitition but okay I guess it's good enough this way ;)